front_end/panels/react_devtools/ReactDevToolsModel.ts

```diff
@@ -31,91 +30,163 @@ type ReactDevToolsBindingsBackendExecutionContextUnavailableEvent = Common.Event
 
 export class ReactDevToolsModel extends SDK.SDKModel.SDKModel<EventTypes> {
   private static readonly FUSEBOX_BINDING_NAMESPACE = 'react-devtools';
-  private readonly rdtBindingsModel: ReactNativeModels.ReactDevToolsBindingsModel.ReactDevToolsBindingsModel | null;
+
+  readonly #wall: ReactDevToolsTypes.Wall;
+  readonly #bindingsModel: ReactNativeModels.ReactDevToolsBindingsModel.ReactDevToolsBindingsModel;
+  readonly #listeners: Set<ReactDevToolsTypes.WallListener> = new Set();
+  #initialized: boolean = false;
```

Comment:
TypeScript supports `private` keyword, and it is far more popular than `#` in this codebase. I don't have a strong opinion on this, but would prefer sticking to a single style across the whole codebase.

Is there a specific reason to use `#` over `private`?

Reply:
Yeah, functionally `#` will act as private members at runtime, added to JS later than TypeScript's `private`. We definitely don't need to replace everywhere, but I'd prefer this convention in RN-unique files.